webapp/src/browser_routing.js

```diff
@@ -0,0 +1,9 @@
+const WebappUtils = window.WebappUtils;
+
+const navigateToUrl = (urlPath) => {
+    WebappUtils.browserHistory.push(urlPath);
+};
+
+export const navigateToChannel = (team, channel) => {
+    navigateToUrl(`${team}/messages/@${channel}`);
```

Comment:
In general, it's better to prefix with a trailing slash. Otherwise, depending on the route and the router set up, push ends up appending to the existing route, which is not desired.